### PR TITLE
Facebook Pixel doc fix

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> Add a note that `revenue` is a required field for the events mapped to the `Purchase` standard event.

Context: [This Slack thread](https://rudderstack.slack.com/archives/C01E4PLB135/p1675426990331889?thread_ts=1675257762.310809&cid=C01E4PLB135)

## Type of documentation

- [ ] New documentation
- [ ] Documentation update
- [x] Hotfix

### CODE REVIEW

File: docs/destinations/streaming-destinations/fb-pixel.mdx [hunk 1]
```diff
@@ -279,6 +279,7 @@ You can use the <strong>Map your events with Facebook Standard Events</strong> s
 
 Note that:
 
+- For the `Purchase` standard event, `revenue` is a required field.
 - For the <code class="inline-code">Products Searched</code> event, the <code class="inline-code">query</code> property must be a string.
 - You need to specify `currency` in the `Order Completed` event. If not specified, RudderStack will set the default value to `USD`.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/fb-pixel.mdx:284]: @shrouti1507 please review if this is a required field, and if we can rephrase this to:

```
If you do not specify the `currency` field in the `Order Completed` event, RudderStack sets its default value to `USD`.
```

Comment (shrouti1507) [reply docs/destinations/streaming-destinations/fb-pixel.mdx:284]: yes we can rephrase this.

File: docs/destinations/streaming-destinations/fb-pixel.mdx [hunk 2]
```diff
@@ -279,6 +279,7 @@ You can use the <strong>Map your events with Facebook Standard Events</strong> s
 
 Note that:
 
+- For the `Purchase` standard event, `revenue` is a required field.
```
Comment (shrouti1507) [inline docs/destinations/streaming-destinations/fb-pixel.mdx:282]: we need to write `properties.revenue`.

Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/fb-pixel.mdx:282]:
```suggestion
- For the `Purchase` standard event, `properties.revenue` is a required field.
```

File: docs/destinations/streaming-destinations/fb-pixel.mdx (hunk 1 above)
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/fb-pixel.mdx:284]:
```suggestion
- If you do not specify the `currency` field in the `Order Completed` event, RudderStack sets its value to `USD` by default.
```

File: docs/destinations/streaming-destinations/fb-pixel.mdx [hunk 3]
```diff
@@ -279,8 +279,9 @@ You can use the <strong>Map your events with Facebook Standard Events</strong> s
 
 Note that:
 
+- For the `Purchase` standard event, `properties.revenue` is a required field.
 - For the <code class="inline-code">Products Searched</code> event, the <code class="inline-code">query</code> property must be a string.
-- You need to specify `currency` in the `Order Completed` event. If not specified, RudderStack will set the default value to `USD`.
+- If you do not specify the `currency` field in the `Order Completed` event, RudderStack sets its value to `USD` by default.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/fb-pixel.mdx:284]:
```suggestion
- If you do not specify the `properties.currency` field in the `Order Completed` event, RudderStack sets its value to `USD` by default.
```

**Review by ishachopra0401 — APPROVED** [review]

**Review by shrouti1507 — APPROVED** [review]
